tests/general/mmap.c

```diff
@@ -0,0 +1,87 @@
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <limits.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#ifdef __GLIBC__
```

Comment:
Is there a requirement that these tests build outside of wasi-sdk too?  Do we have a test runner mode that does that?

Reply:
We don't have that right now, though we may want that in the future.